spec/support/capybara.rb

```diff
@@ -7,6 +7,20 @@
 # hidden elements are ignored by default
 # Capybara.ignore_hidden_elements = true
 
+client = Selenium::WebDriver::Remote::Http::Default.new
+client.read_timeout = 120 # instead of default 60, in seconds
```

Comment:
that seems a bit insane.. that means each test can take 2 minutes ?

Reply:
is this the part that fixes the `IO::EAGAINWaitReadable` issue?

Reply:
Each feature spec failure can take up to 2 mins until it generates an `Net::ReadTimeout` exception (which is from the Net::Http client), the `IO::EAGAINWaitReadable` is the underlying exception caught and wrapped in a `Net::ReadTimeout`. Making the app faster would obviate the need for this, it occurs on the first hit to the webserver during a feature spec, usually the sign in page.